tools/bisect-git-annex

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# A helper script which would take actual bisection script to be executed.
+# It will first build git-annex using Singularity image, assuming that
+# to bisect git-annex using a build environment in singularity image
+#
+set -eu
+SINGULARITY_IMAGE="$1"
+# The rest of cmdline is to execute
+shift
+
+if [ ! -e "$SINGULARITY_IMAGE" ]; then
+    echo "I: trying to fetch singularity image with build env"
+    singularity pull --name "$SINGULARITY_IMAGE" shub://datalad/datalad:git-annex-dev
+fi
+
+SINGULARITY_IMAGE=$(readlink -f "$SINGULARITY_IMAGE")
+annex_gitver=$(git describe)
+logfile=$PWD/.git/logs/git-annex-build-$annex_gitver.log
```

Comment:
Probably doesn't matter in practice, but it seems inappropriate to store the log in git's reflog directory.  Why not use a temporary path?

Reply:
things get lost in temporary path since then you need to have a temporary filename etc.
but indeed, I thought this `logs/` it is more generic than reflogs -- just curious what else could there be other than HEAD?
anyways -- will now RF to use `mktemp -p '' git-annex-build-${annex_gitver}-XXXXXX.log` instead, testing now and will push.
With that I will consider this PR done - it already served its purpose ;-)